pkg/interface/contacts/src/js/components/lib/contact-card.js

```diff
@@ -499,38 +499,36 @@ export class ContactCard extends Component {
     //TODO "Share card" if it's /me -> sends to /~/default of recipient
     return (
       <div className="w-100 h-100 overflow-hidden">
-        <div className="w-100 bg-white bb b--gray4">
+        <div className="flex justify-between w-100 bg-white bb b--gray4 ">
           <div className="w-100 h2 dn-m dn-l dn-xl inter pb6 pl3 pt3 f8">
             <Link to="/~contacts/">{"⟵"}</Link>
           </div>
-          <button
-            onClick={() => {
-              if (props.share) {
-                this.shareWithGroup();
-              } else {
-                this.editToggle();
-              }
-            }}
-            className={`pl4 mt4 mb4 f9 pointer b--black ` + ourOpt}>
-            {editInfoText}
-          </button>
-          <button className={`pl4 mt4 mb4 f9 b--black ` + localOpt}>
-            Share Contact Info
-          </button>
+          <div className="flex">
+            <button
+              onClick={() => {
+                if (props.share) {
+                  this.shareWithGroup();
+                } else {
+                  this.editToggle();
+                }
+              }}
+              className={`ml3 mt2 mb2 f9 pa1 ba br2 pointer b--black ` + ourOpt}>
```

Comment:
Why did you put `ba` back into these? I don't think this is OS1 in mock.

Reply:
These were buttons the last time I synced with the os1-rc branch. I can remove these.

Reply:
You might want to do it like

```
git checkout os1-rc
git pull
git checkout jy/copy
git merge os1-rc
```

I don't want to have regressions in UI from this, basically...

Reply:
Though it may just be this one thing that is sticking around.

Reply:
I personally adore the look of ba + br styling in Urbit, but afaik we're keeping them unstyled per @matildepark 's obervation

Reply:
Per OS1 mocks:

![image](https://user-images.githubusercontent.com/20846414/75193634-9ba5e080-5724-11ea-8fd0-7e0506c1bcb3.png)

If you want to change that, I think we should probably be consistent with it, but I think this is ultimately design purview, @urcades.

Reply:
No, let's not change these things, just keep them unstyled text for the time being.